installation-bundle/src/Config/ParameterDumper.php

```diff
@@ -78,9 +87,17 @@ public function dump(): void
         }
 
         $this->filesystem->dumpFile(
-            $this->rootDir.'/app/config/parameters.yml',
+            $this->rootDir.'/config/parameters.yml',
             "# This file has been auto-generated during installation\n".Yaml::dump($this->getEscapedValues())
         );
+
+        if ($this->filesystem->exists($this->rootDir.'/app/config/parameters.yml')) {
+            $this->filesystem->remove($this->rootDir.'/app/config/parameters.yml');
```

Comment:
What about the `.dist` file?

Reply:
Well we can't just delete it, we would need to move it as well. And I don't want to move around user files.